lib/hyacinth/adapters/external_identifier_adapter/hyacinth_metadata.rb

```diff
@@ -22,33 +28,21 @@ def doi
   end
 
   def created_at
-    deserialize_datetime(@source['created_at'])
+    DATETIME_PARSER.from_serialized_form_impl(@source['created_at'])
   end
 
   def updated_at
-    deserialize_datetime(@source['updated_at'])
+    DATETIME_PARSER.from_serialized_form_impl(@source['updated_at'])
   end
 
   def first_published_at
-    deserialize_datetime(@source['first_published_at'])
+    DATETIME_PARSER.from_serialized_form_impl(@source['first_published_at'])
   end
```

Comment:
Why are these methods necessary if the `initialize` method is given an an already-deserialized digital object?

Reply:
I looked at the constructor and answered my own question.  We can simplify these methods (and get rid of the reference to the TypeDef, which we probably don't want to tie to this class anyway) if we don't serialize the digital object to JSON.

Reply:
This is a good point - in this (and a couple of other spots in your review) you are seeing refactors of existing implementation that you are suggesting further refactors to, not original implementations.